lib/screens/create_region_screens/review_region_information.dart

```diff
@@ -33,7 +33,11 @@ class ReviewRegion extends StatelessWidget {
               );
             },
           );
+        } else if (pageCommand is NavigateToNewRegion) {
```

Comment:
I think we have a generic NavigateTo PageCommand, or do you need to make another logic in addition to navigation?

Reply:
Generic one required an argument string, I will use it on next pr when I know what logic needs to be place here